.github/workflows/lint.yml

```diff
@@ -0,0 +1,35 @@
+name: Lint
```

Comment:
Does this need License header as well?

Reply:
I believe that's done in the last few lines of the file